lib/ssh_scan/api_db.rb

```diff
@@ -0,0 +1,61 @@
+require 'sqlite3'
+
+module SSHScan
+  class APIDatabaseHelper
+    def initialize(database_name)
+      if File.exist?(database_name)
+        @db = ::SQLite3::Database.open(database_name)			
+      else
+        @db = ::SQLite3::Database.new(database_name)
+        self.create_schema
+      end
+    end
+
+    def create_schema
+      @db.execute <<-SQL
+        create table api_schema (
+          worker_id varchar(100),
+          uuid varchar(100),
+          result json
```

Comment:
Not too sure if json is a datatype with SQLite3 that Ruby has (It is a datatype in SQLite 3.9). Did you want to do varchar or longtext here?